README.md

```diff
@@ -12,12 +12,47 @@ This method is considered advanced and should only be used if one is an expert i
 
 ## Installation
 
-Run as root (sudo su):
+Section 1 provides links to download Debian 10 and information on installation. If you already have Debian 10 installed and wish to move on to installing Home Assistant, proceed to Section 2. 
+
+### Section 1 – Install Debian 10
+
+**1.1)** Start by downloading `debian-live-10.5.0-amd64-standard.iso.torrent` from [HERE](https://cdimage.debian.org/debian-cd/current-live/amd64/bt-hybrid/). This is a torrent file, so use your favourite torrent program. Alternitively, you can download the `debian-live-10.5.0-amd64-standard.iso` ISO from [HERE](https://cdimage.debian.org/debian-cd/current-live/amd64/iso-hybrid/).
+
+Some useful infomation on how to install Debian 10 can be found on the Debain website [HERE](https://www.debian.org/releases/stable/installmanual)
```

Comment:
This is still out of scope. Instead, we should list the requirements. Also, linking version-specific downloads is prone to run out of date rather quickly. Before you know it, we will be recommending an outdated download.